src/sentry/static/sentry/app/actionCreators/projects.jsx

```diff
@@ -93,38 +93,96 @@ export function transferProject(api, orgId, project, email) {
 /**
  * Associate a team with a project
  */
-export function addTeamToProject(api, orgSlug, projectSlug, teamSlug) {
-  let endpoint = `/projects/${orgSlug}/${projectSlug}/teams/${teamSlug}/`;
+
+/**
+ *  Adds a team to a project
+ *
+ * @param {Client} api API Client
+ * @param {String} orgSlug Organization Slug
+ * @param {String} projectSlug Project Slug
+ * @param {String} teamSlug Team Slug
```

Comment:
It's `team` now?

Reply:
For remove team it's still slug -- I'm not sure if we always have full `team` in scope